src/views/Portal/components/ApplicationForm/PriorExperience/index.view.tsx

```diff
@@ -10,50 +10,48 @@ const ExperiencePage: React.FC = () => {
     useApplication()!
   return (
     <div className='experience-page'>
-      <div className='experience-page__container'>
```

Comment:
We need the `flex-direction: column;`, or it'll look like this:

<img src="https://user-images.githubusercontent.com/20177171/140622311-49df0e5b-07b0-44c1-bd86-c590b72a57d9.png" width="500px" />